crates/author-ext-api/src/lib.rs

```diff
@@ -17,7 +17,7 @@ pub enum CreateSignedSetKeysExtrinsicError {
 
 sp_api::decl_runtime_apis! {
     /// Runtime API for the author extension logic.
-    pub trait AuthorExtApi<Id: Encode> {
+    pub trait AuthorExtApi<Id> where Id: Encode {
```

Comment:
This diff looks like it doesn't belong here

Reply:
The changes have been applied after merging master containing updated `rust-toolchain`. Clippy noted an error  here before.

Reply:
<img width="758" alt="image" src="https://github.com/user-attachments/assets/e010d29f-c1da-4780-a761-ea76777896ef">

Reply:
Why is it in this diff? Should be fixed in `master`

Reply:
Clippy doesn't note it in master.

Reply:
It must be a change in the macro then... But actually what clippy says here is that you can omit the `Encode` requirement as it is defined by the macro now

Reply:
Yep, but the interesting fact that if I omit the `Encode` that I have build error )

Reply:
Why did it only appear with the last rebase though?